package.json

```diff
@@ -18,6 +18,7 @@
     "04": "echo -------------------------------------------- Demo Shell -----------------------------------------------",
     "start": "nx serve demoshell --open",
     "start:prod": "nx serve demoshell --configuration production --open",
+    "update-webdriver": "node runany.js update-webdriver",
```

Comment:
there is no `runany.js` file in ADF

Reply:
this I added just to run e2e as, it was not able to run due no affected changes dictated.  I will remove this. 
here we can see e2e test able to run successfully. https://github.com/Alfresco/alfresco-ng2-components/actions/runs/9253234281/job/25453146522